test/sanitize-html.test.js

```diff
@@ -6,3 +6,57 @@ describe('Sanitize HTML', () => {
     expect(data).toBe('<img src="x" />');
   });
 });
+
+describe('Sanitize HTML', () => {
+  test('<a><img> should work, but inline js should not', () => {
+    const data = sanitizeHTML(
+      '<a href="https://www.telescope.com"><img border="0" alt="W3Schools" src="logo.gif" width="100" height="100"></a>'
+    );
+    expect(data).toBe('<a href="https://www.telescope.com"><img src="logo.gif" /></a>');
+  });
+});
+
+describe('Sanitize HTML', () => {
+  test('<p> should work with inline style', () => {
+    const data = sanitizeHTML('<p style="color:blue;font-size:46px;">Here is color blue</p>');
+    expect(data).toBe('<p>Here is color blue</p>');
+  });
+});
+
+describe('Sanitize HTML', () => {
+  test('<p> and <strong> should work if there are two tab in the same line', () => {
+    const data = sanitizeHTML(
+      '<p style="color:blue;font-size:46px;">Here is color <strong>blue</strong></p>'
+    );
+    expect(data).toBe('<p>Here is color <strong>blue</strong></p>');
+  });
+});
+
+describe('Sanitize HTML', () => {
+  test('<figure> with <img> should remove <figure>', () => {
```

Comment:
I wonder if we should add `<figure>` to the list of allowed elements?  Seems useful.  Can you do that?

Reply:
ok, i will file a new issue to add ```<figure>```